src/platform_impl/windows/window.rs

```diff
@@ -3,31 +3,62 @@
 use parking_lot::Mutex;
 use raw_window_handle::{windows::WindowsHandle, RawWindowHandle};
 use std::{
-    cell::Cell,
+    cell::{Cell, RefCell},
     ffi::OsStr,
     io, mem,
-    os::windows::ffi::OsStrExt,
+    os::windows::prelude::OsStrExt,
```

Comment:
You changed this from `ffi` to `prelude`, should the same happen in the `THREAD_EVENT_TARGET_WINDOW_CLASS` constructor?

Reply:
Don't know, this was probably accidentally by rust-analyzer.